requirements.txt

```diff
@@ -21,7 +21,7 @@ pandas==0.24.2
 pdfkit==0.6.1
 ply==3.11
 pygraphviz==1.5
-Pyomo==5.7
+Pyomo==5.6.6
```

Comment:
Why does that have to be changed? Just curious!

Reply:
So, I remembered from Offgridders that it was not possible to use the newest pyomo version for oemof-solph, and I got an pyomo related error... but I tried it again just now and it works. So I guess that it can be v. 5.7 after all.